src/components/DateRangeOutput/index.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { _cs } from '@togglecorp/fujs';
+
+import DateOutput, { Props as DateOutputProps } from '../DateOutput';
+import styles from './styles.css';
+
+export interface Props {
+    className?: string;
+    startDate: DateOutputProps['value'];
+    endDate: DateOutputProps['value'];
```

Comment:
@frozenhelium we need to handle cases where there are no startDate or no EndDate or both are absent.